.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
let's make sure we note Python 3.7+ in right places (readme/docs etc.)

Reply:
I don't think dropping the 3.6 testing _necessarily_ means that we say the library is 3.7+. It still works on 3.6 - we just can't test it automatically. Would you rather be on the safer side and drop it altogether?

Reply:
oh I see.. I got confused with Seq2Pat, as in that case, 3.6 does not work, hence dropped fully jumping to 3.7 in the docs/readme/requirements. In this case, I leave it you then,

Reply:
I'm inclined to keep 3.6 until we get a report saying that 3.6 is causing issues.